test/spectest.main.v3

```diff
@@ -37,22 +41,23 @@ component spectest {
 			OUT.putd(count);
 			OUT.ln();
 		}
+                var result = 0;
 		for (a in args) {
 			if (a == null) continue;
-			if (Strings.endsWith(a, ".wast")) runTest(engine, a);
+                        if (Strings.endsWith(a, ".wast")) result += runTest(engine, a);
```

Comment:
POSIX only reports the low 8 bits of an exit code, so to prevent wraparound, this should probably do a `|=`.